rnndb/vdec/vp3/pppp.xml

```diff
@@ -64,11 +64,11 @@
 		</reg32>	<!-- for surface layout purposes -->
 		<reg32 offset="0x08" name="POUT_CONFIG">
 			<bitfield low="0" high="2" name="LAYOUT">
-				<value value="0" name="LINEAR_RAW"/> <!-- like LINEAR, but SURF_WIDTH is ignored and PIC_WIDTH is used instead -->
```

Comment:
Careful with these - PPPP formats aren't exactly aligned with the rest of the GPU.

Change PITCH_RAW to LINEAR - it basically behaves like 1D.